CHANGELOG.md

```diff
@@ -5,6 +5,36 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v3.47.0](https://github.com/buildkite/agent/tree/v3.47.0) (2023-05-23)
+[Full Changelog](https://github.com/buildkite/agent/compare/v3.46.1...v3.47.0)
+
+Two new and very noteworthy experiments!
+
+1. Have you ever wanted to write hooks in a compiled language? Or in Python or
+   Ruby? Well now you can! With `--experiment=polyglot-hooks` the agent can run
+   all sorts of hooks and plugins directly. Combined with 
+   `--experiment=job-api`, your hooks-of-a-different-language can alter
+    environment variables through the local Job API!
+2. Concurrency groups are great, but have you ever wanted to manage multiple
+   agents running on the same host concurrently accessing a shared resource? 
+   Well now you can! With `--experiment=agent-api`, the agent now has an inbuilt
+   locking service, accessible through new `lock` subcommands and also via a
+   Unix socket (like the `job-api`).
+
+### Added
+- Experiment: Polyglot hooks [#2040](https://github.com/buildkite/agent/pull/2040) (@moskyb)
+- Experiment: Local Agent API, with locking service [#2042](https://github.com/buildkite/agent/pull/2042) (@DrJosh9000)
+- New flag `--upload-skip-symlinks` (on `artifact upload`) allows skipping symlinks when uploading files. `--follow-symlinks` has been deprecated and renamed to `--glob-resolve-follow-symlinks` [#2072](https://github.com/buildkite/agent/pull/2072) (@triarius)
+
+### Fixed
+- The `normalised-upload-paths` experiment was unintentionally left out of the available experiments list [#2076](https://github.com/buildkite/agent/pull/2076) (@MatthewDolan)
+- The `--no-patch` flag passed to git is replaced with `-s`, to continue supporting older versions of git (pre v1.8.4; 2013) [#2075](https://github.com/buildkite/agent/pull/2075) (@pda)
```

Comment:
Hmm, this is in the changelog for v3.46.1 already. 
```suggestion
```
Maybe there's some technicality about the git history causing this to appear here, but I think as we released v3.46.1 before this version, we should treat v3.46.1 as an ancestor of this version even if the git commits are not.

Reply:
Yep, I just missed it. Done.